src/services/gateways/aws.ts

```diff
@@ -154,7 +154,8 @@ import {
   Route53Client
 } from '@aws-sdk/client-route-53'
 import { IAM } from '@aws-sdk/client-iam'
-import { RecordType } from '../../modules/aws_route53_hosted_zones@0.0.1/entity/resource_records_set'
+// WTF? Why is this circular reference here?
+import { RecordType } from '../../modules/0.0.1/aws_route53_hosted_zones/entity/resource_records_set'
```

Comment:
Oh probably this was the auto import, RecordType probably should come from an AWS lib.